sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -136,18 +129,7 @@ public EventData(BinaryData body) {
      * @return Application properties associated with this {@link EventData}.
      */
     public Map<String, Object> getProperties() {
-        return properties;
-    }
-
-    /**
-     * Properties that are populated by Event Hubs service. As these are populated by the Event Hubs service, they are
-     * only present on a <b>received</b> {@link EventData}.
-     *
-     * @return An encapsulation of all system properties appended by EventHubs service into {@link EventData}.
-     *     {@code null} if the {@link EventData} is not received from the Event Hubs service.
-     */
-    public Map<String, Object> getSystemProperties() {
```

Comment:
This is a break change. We should keep public APIs

Reply:
According to your comment, fixed within new version.